### PR TITLE
Bluetooth: Classic: Add length check of user input

### PR DESCRIPTION
**Pull request by ekleezg**
Added a check for the length of user input as it was missing.

### CODE REVIEW

File: subsys/bluetooth/host/classic/sdp.c [hunk 1]
```diff
@@ -1750,6 +1750,11 @@ static int sdp_client_receive(struct bt_l2cap_chan *chan, struct net_buf *buf)
 
 	switch (hdr->op_code) {
 	case BT_SDP_SVC_SEARCH_ATTR_RSP:
+		/* Check the buffer len for the length field */
+		if (buf->len < 2) {
```
Comment (jhedberg) [inline subsys/bluetooth/host/classic/sdp.c:1754]: Just for consistency, I suppose `sizeof(uint16_t)` would be better than `2`. Otherwise looks good.

Comment (ekleezg) [reply subsys/bluetooth/host/classic/sdp.c]: Good. I changed it.

**Comment by jhedberg** [comment]
@ekleezg thanks, but no need to split these into multiple commits since they're all quite related to each other. Just squash the changes into a single commit and force push. Also add a commit message body (i.e. some additional text than just the commit subject), since the Compliance CI check will otherwise fail.

**Review by Thalley — APPROVED** [review]

**Review by lylezhu2012 — APPROVED** [review]

**Review by jhedberg — APPROVED** [review]

**Comment by kartben** [comment]
@ekleezg you need to amend your git commit author name/email so that it matches the sign-off entry. Thanks!

**Comment by ekleezg** [comment]
Thank you for let me know. I changed it.

**Pull request merged**